### ISSUE TITLE
afterRender group of components

### ISSUE THREAD

**Issue by sunprophit** [knockout/knockout#1906]
Is it possible to call some afterRender binding after group or all components in a view have been rendered? template afterRender or custom binding can't solve this problem: both of them could be invoked before component would be rendered.

**Comment by wind-raven** [comment]
#1856 has provided this and has a pull request pending.  Hopefully it gets a little more traction.

**Comment by brianmhunt** [comment]
Yep, this is a dupe. Thanks @sunprophit for the report, and @wind-raven for pointing to the ongoing PR.  Cheers.
